s3-import-commons/src/main/java/no/unit/nva/publication/s3imports/FileEntriesEventEmitter.java

```diff
@@ -140,17 +159,18 @@ private UriWrapper generateErrorReportUri(ImportRequest input) {
     }
 
     private List<PutEventsResult> generateReportIndicatingTotalEmissionFailure(
-        Try<List<PutEventsResult>> completeEmissionFailure) {
+        Try<List<PutEventsResult>> completeEmissionFailure, String s3Location) {
         PutEventsResponse customPutEventsResponse =
-            generatePutEventsResultIndicatingNoEventsWereEmitted(completeEmissionFailure);
+            generatePutEventsResultIndicatingNoEventsWereEmitted(completeEmissionFailure, s3Location);
         PutEventsResult putEventsResult = new PutEventsResult(NO_REQUEST_WAS_EMITTED, customPutEventsResponse);
         return Collections.singletonList(putEventsResult);
     }
 
     private PutEventsResponse generatePutEventsResultIndicatingNoEventsWereEmitted(
-        Try<List<PutEventsResult>> completeEmissionFailure) {
+        Try<List<PutEventsResult>> completeEmissionFailure, String s3Location) {
         String exceptionStackStraceAsResultMessage =
-            stackTraceInSingleLine(completeEmissionFailure.getException());
+            "File in location: " + s3Location + ". Failed with the following exception: "
```

Comment:
psfs  EXCEPTION_STACKTRACE_MESSAGE_TEMPLATE = ""File in location: %s Failed with the following exception:  %s";
String.format(EXCEPTION_STACKTRACE_MESSAGE_TEMPLATE, s3Location, stacktraceInSingleLine(…));